spaces/domains.json

```
@@ -217,5 +217,6 @@
   "vote.moonpirate.finance": "moonpirate.eth",
   "vote.mai.finance" : "qidao.eth",
   "dao.modefi.io" : "mod.eth",
-  "vote.koaladefi.eth": "koaladefi.eth"
+  "vote.koaladefi.eth": "koaladefi.eth",
+  "cybertime.finance": "cybertime.finance"
```

You will need to use a subdomain and your ENS.
```suggestion
  "vote.cybertime.finance": "cybertime.eth"
```